package.json

```diff
@@ -25,7 +25,12 @@
     "build": "tsc",
     "lint": "eslint --ext=.ts src",
     "test": "jest",
-    "test:lambda": "yarn build && lambda-local -l ./dist/index.js -e ./dist/mock-events/default -h randomGet"
+    "test:lambda": "yarn build && lambda-local -l ./dist/index.js -e ./dist/mock-events/default -h randomGet",
```

Comment:
Keep in mind that with the directory restructuring, the paths where those files previously resided should be double checked as well.